www/less/misc.less

```diff
@@ -82,3 +82,26 @@ a{
 * {
     -webkit-touch-callout: none !important;
 }
+
+//updating typography for h2 and h3
+.content h2 {
+    padding: 20px 0 10px;
+    border-bottom: 1px dotted #eee;
```

Comment:
The dots are causing funky times for some section heading if the section heading text extends almost to the right edge of the screen. See `enwiki > History of Apple Inc > 1969-1984: Jobs and Wozniak` and other sections on that article.

If you want to just remove the dots for now until we can iron that out, I'd be fine merging the other changes if you want to do a quick commit to fix the other issues I noted.